content/doc/book/pipeline/syntax.adoc

```diff
@@ -306,15 +306,17 @@ pipeline {
     agent none // <1>
     stages {
         stage('Example Build') {
-            agent { docker: 'maven:3-alpine' } // <2>
+            agent { docker 'maven:3-alpine' } // <2>
             steps {
-                sh 'mvn -B clean verify'
+                echo 'Hello, Maven'
+                sh 'mvn --version'
```

Comment:
Not hugely important, but I changed this example so that the build doesn't fail (due to absence of a checked-out POM).